src/pages/Account/Profile.tsx

```diff
@@ -60,18 +55,12 @@ export const Profile: React.FC = () => {
 
       await updateUserProfile(user.uid, data);
 
-      const accountQuestionsUrlParams = new URLSearchParams();
-      accountQuestionsUrlParams.set("spaceSlug", spaceSlug);
-      returnUrl && accountQuestionsUrlParams.set("returnUrl", returnUrl);
-
       // @debt Should we throw an error here rather than defaulting to empty string?
-      const nextUrl = spaceSlug
-        ? `${ACCOUNT_PROFILE_QUESTIONS_URL}?${accountQuestionsUrlParams.toString()}`
-        : returnUrl ?? "";
+      const nextUrl = accountProfileQuestionsUrl(worldSlug, spaceSlug);
```

Comment:
!!!helper!!! 

Check should be encapsulated inside the helper function (i.e. default space slug or `"#"` as a result or something else)
OR
just not create new function solely to make `generatePath()` parameters ordered